sympy/core/tests/test_relational.py

```diff
@@ -662,6 +662,33 @@ def test_issue_10304():
     e = 1 + d*I
     assert simplify(Eq(e, 0)) is S.false
 
+
+def test_issue_10401():
+    fin = symbols('inf', finite=True)
+    inf = symbols('inf', infinite=True)
+    inf2 = symbols('inf2', infinite=True)
+    zero = symbols('z', zero=True)
+    nonzero = symbols('nz', zero=False, finite=True)
+    E = Eq
+    T, F = S.true, S.false
```

Comment:
@smichr not need to add same test cases for both because ```Ne``` also call ```Eq``` to check condition. need to (4-5) test cases for ```Eq``` and (4-5) test cases for ```Ne```(in both put different test cases and cover all below test cases).